Classes/Notifications/NotificationModelController.swift

```diff
@@ -24,6 +24,14 @@ extension NotificationViewModel {
     }
 }
 
+// sorting Array<InboxDashboardModel> by date for mentioned filter type
+extension Array where Element == InboxDashboardModel {
```

Comment:
It would also be 💯 if we can add a test for this one!

Reply:
I added tests for this functionality.